prometheus_hardware_exporter/__main__.py

```diff
@@ -41,6 +41,11 @@ def main() -> None:
     for name, collector in COLLECTOR_REGISTRIES.items():
         if name.lower() in enabled_collectors:
             logger.info("collector %s is enabled", name)
+            if name.lower() == "redfish-collector":
+                exporter.register(
+                    RedfishCollector(config.bmc_host, config.bmc_username, config.bmc_password)
+                )
+                continue
```

Comment:
I prefer to use if/else to show the logic

Reply:
Will collector read the config file internal? If yes, then we don't need to parse the username/password here. 

Another idea is `config` will be a parameter for all collecter.
if config == None => use default value
if config != None => collector will change the behavior.

Reply:
I guess you misunderstand what I mean based on the latest source code. 
I didn't mean to unzip the collector table. I just say not to use `continue`
```python
    for name, collector in COLLECTOR_REGISTRIES.items():
        if name.lower() in enable_collectors_set:
            logger.info("%s is enabled", name)
            if name.lower() == "collector.redfish":
                  exporter.register(
                    RedfishCollector(config.bmc_host, config.bmc_username, config.bmc_password)
                )
            else:
                  exporter.register(collector)
```

Reply:
Oh I got it. I thought you wanted to make the `registration` consistent for all collectors. Anyway, in the next commit(after lint fix), I am using config directly in the collector, so this issue will go away.